<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BORealTimeChartSettingsDialog</class>
 <widget class="QDialog" name="BORealTimeChartSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="showGroupBox">
        <property name="minimumSize">
         <size>
          <width>127</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>127</height>
         </size>
        </property>
        <property name="title">
         <string notr="true">Show</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="gridCheckBox">
             <property name="text">
              <string notr="true">Grid</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="barStrokeCheckBox">
             <property name="text">
              <string notr="true">BarStroke</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="crosshairCheckBox">
             <property name="text">
              <string notr="true">Crosshair</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="colorsGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>262</height>
         </size>
        </property>
        <property name="title">
         <string notr="true">Colors</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="gridLabel">
               <property name="text">
                <string notr="true">Grid:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="gridColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="fgLabel">
               <property name="text">
                <string notr="true">Foreground:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="fgColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="bgLabel">
               <property name="text">
                <string notr="true">Background:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bgColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="coorBgLabel">
               <property name="text">
                <string notr="true">Coordinate Background:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="coorBgColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="barStrokeLabel">
               <property name="text">
                <string notr="true">Bid:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="barStrokeColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="crosshairLabel">
               <property name="text">
                <string notr="true">Crosshair:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="crossColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="bullishLabel">
               <property name="text">
                <string notr="true">Bullish:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bullishColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="bearishLabel">
               <property name="text">
                <string notr="true">Bearish:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bearishColorBT">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>131</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string notr="true">OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string notr="true">Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>BORealTimeChartSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>BORealTimeChartSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
